package.json

```diff
@@ -106,11 +106,11 @@
         "fpscanner": "^0.1.5",
         "jest": "^26.6.3",
         "jest-circus": "^26.6.3",
-        "playwright": "1.10.0",
+        "playwright": "1.7.1",
```

Comment:
any reason to downgrade those? obviously it fails to install on m1 again due to this.

Reply:
Yeah, needed the release to finish and Chrome 90 is still not out as stable. You can bump them back again now.